### PR TITLE
Update default role policy

### PR DESCRIPTION
**Pull request by alisalahio**
closes https://github.com/serverless/components/issues/918#issuecomment-814418217

### CODE REVIEW

File: serverless.component.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: aws-lambda
-version: 2.0.2
+version: 3.0.0
```
Comment (MattiaPette) [inline serverless.component.yml:2]: I think this should not be a major release, maybe a patch is a better fit?
```suggestion
version: 2.0.3
```

**Review by MattiaPette — CHANGES_REQUESTED** [review]

**Comment by alisalahio** [comment]
@MattiaPette I'm not sure, but this is [how the express component did it](https://github.com/serverless-components/express/pull/70/commits/3f6ca759c2ee5121386477398d1f67a00ca3b05b), possibly because it's a breaking change.

Are you sure I should change it?

**Comment by joshvtech** [comment]
Any idea how long this PR could take? Can this be fixed locally whilst we wait?

**Comment by janpio** [comment]
This currently leads to the normal "Deploy Lambda" getting started project on https://app.serverless.com/ being broken :/

Update: As a hack I published `component: aws-lambda-fixed` based on this PR and this seems to allow me to deploy to Lambda.

**Comment by mthan160** [comment]
> This currently leads to the normal "Deploy Lambda" getting started project on https://app.serverless.com/ being broken :/
> 
> Update: As a hack I published `component: aws-lambda-fixed` based on this PR and this seems to allow me to deploy to Lambda.

Yes I can confirm that, it's broken because of this.

I tried tearing apart the npm package `serverless` and couldn't find any references to `AWSLambdaFullAccess` - where can we make the changes suggested in this PR?

**Comment by janpio** [comment]
If you are using that example, is says `component: aws-lamda` in the `serverless.yml`. Components seem to be downloaded on runtime, and not part of your node_modules if I got that right - which is why I published the `aws-lambda-fixed` package to that serverless components registry. (But I am also super green in all of this, so 🤷 if this is the right way to do this)

**Comment by LukeFerris** [comment]
@janpio - thanks! I'm a Serverless beginner and there's nothing quite like following the default flow and it throwing an error you don't understand. I changed the serverless.yml to your update and it deployed successfully. I am in your debt.

If I've understood correctly, this is a breaking bug for all new lambda components? (including new, easily confused people like me). Is there some way we can increase the priority for the merge?

**Review by eahefnawy — APPROVED** [review]
Thanks for this fix @alisalahio! 🙌

**Comment by eahefnawy** [comment]
This is now published! Thanks all!

**Pull request merged**